database_handler.py

```diff
@@ -0,0 +1,138 @@
+import mysql.connector
+import os
+from mysql.connector import errorcode
+
+
+class DatabaseHandler:
+    def __init__(self):
+        self.config = {
+            "user": os.getenv("DB_USER"),
+            "password": os.getenv("DB_PASSWORD"),
+            "host": os.getenv("DB_HOST"),
+            "port": int(os.getenv("DB_PORT")),
+            "database": os.getenv("DB_NAME"),
```

Comment:
These values are read from the `.env` file. To make it clear that these are needed, let's add some examples to `.env.dist`:

```
DB_HOST=localhost
DB_PORT=3306
DB_NAME=scrapyd_stats
DB_USER=my_username
DB_PASSWORD=my_secret_password
```